### PR TITLE
Feat(eos_cli_config_gen): support for mlag peer-link requests disabled under dhcp_relay

### LINKED ISSUE #3254: Feat(eos_cli_config_gen): add support for mlag peer-link requests disabled under dhcp_relay
### Enhancement summary

From EOS 4.28.0F this option is supported under dhcp_relay to suppress flooding of VTEP decapsulated dhcp packets over the MLAG.
https://www.arista.com/en/support/toi/eos-4-25-1f/14626-head-end-replicated-dhcp-packets-suppression-in-vxlan-network

switch(config)# dhcp relay
switch(dhcp-relay)# mlag peer-link requests disabled

### Which component of AVD is impacted

eos_cli_config_gen

### Use case example

running-config:
```
dhcp relay
  mlag peer-link requests disabled
```

### Describe the solution you would like

```
dhcp_relay:
  servers:
    - <str>
  tunnel_requests_disabled: <bool>
  mlag_peerlink_requests_disabled: <bool>
  ```
  

### Describe alternatives you have considered

use eos_cli

### Additional context

_No response_

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by kmueller68**
## Change Summary

add mlag peer-link requests disabled to dhcp relay

## Related Issue(s)

Fixes #3254

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes

1. modified dhcp_relay.j2
2. add schema in dhcp_relay.yml
3. added molecule test

```
dhcp_relay:
  mlag_peerlink_requests_disabled: <bool>
```

## How to test

```
dhcp_relay:
  mlag_peerlink_requests_disabled: true
```
will render
```
dhcp relay
  mlag peer-link requests disabled
```
### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by ClausHolbechArista — APPROVED** [review]

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/dhcp-relay.j2 [hunk 1]
```diff
@@ -15,4 +15,9 @@ dhcp relay
 {%     elif dhcp_relay.tunnel_requests_disabled is arista.avd.defined(false) %}
    no tunnel requests disabled
 {%     endif %}
+{%     if dhcp_relay.mlag_peerlink_requests_disabled is arista.avd.defined(true) %}
+   mlag peer-link requests disabled
+{%     elif dhcp_relay.mlag_peerlink_requests_disabled is arista.avd.defined(false) %}
+   no mlag peer-link requests disabled
+{%     endif %}
```
Comment (gmuloc) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/dhcp-relay.j2]: is this required? We are not testing for it and looking at EOS CLI it does not appear:


```
ceos2(config)#dhcp relay
ceos2(config-dhcp-relay)#mlag peer-link requests disabled
ceos2(config-dhcp-relay)#show active
dhcp relay
   mlag peer-link requests disabled
   server 10.0.0.1
ceos2(config-dhcp-relay)# no mlag peer-link requests disabled
ceos2(config-dhcp-relay)#show active
dhcp relay
   server 10.0.0.1
```

Comment (kmueller68) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/dhcp-relay.j2]: I just did the same as for the already extisting option:

```
{%     if dhcp_relay.tunnel_requests_disabled is arista.avd.defined(true) %}
   tunnel requests disabled
{%     elif dhcp_relay.tunnel_requests_disabled is arista.avd.defined(false) %}
   no tunnel requests disabled
{%     endif %}
```

Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/dhcp-relay.j2]: I don't think there is any other command that could affect the default (unset) behavior, so we only need the non-default option (true in this case)

Comment (kmueller68) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/dhcp-relay.j2]: Should we then also adjust the existing option `tunnel requests disabled`?

```
KHM-t-dehesw-spline-101__13:51:53>ena
KHM-t-dehesw-spline-101__13:51:54#conf t
KHM-t-dehesw-spline-101__13:51:59(config)#dhcp relay 
KHM-t-dehesw-spline-101__13:52:07(config-dhcp-relay)#tunnel requests disabled 
KHM-t-dehesw-spline-101__13:52:15(config-dhcp-relay)#show ac
dhcp relay
   tunnel requests disabled
KHM-t-dehesw-spline-101__13:52:18(config-dhcp-relay)#no tunnel requests disabled 
KHM-t-dehesw-spline-101__13:52:26(config-dhcp-relay)#
KHM-t-dehesw-spline-101__13:52:27(config-dhcp-relay)#show ac
dhcp relay
KHM-t-dehesw-spline-101__13:52:33(config-dhcp-relay)#
KHM-t-dehesw-spline-101__13:52:33(config-dhcp-relay)#show run all section dhcp relay
dhcp relay
   no tunnel requests disabled
   no mlag peer-link requests disabled
 ```

Comment (gmuloc) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/dhcp-relay.j2]: Hi @kmueller68 - indeed it was there for not much either - I have cleaned it up in my commit I just pushed to your branch.
If the CI passes I will approve and merge. Thanks for the contribution!

Comment (kmueller68) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/dhcp-relay.j2]: Thank you very much

**Review by gmuloc — APPROVED** [review]

**Pull request merged**